fuzz/compile-and-execute/src/main.rs

```diff
@@ -0,0 +1,185 @@
+#![no_main]
+
+use rand::prelude::*;
+
+use std::borrow::Borrow;
+use std::collections::HashMap;
+use std::rc::Rc;
+
+use libfuzzer_sys::fuzz_target;
+
+use clvmr::allocator::Allocator;
+
+use clvm_tools_rs::classic::clvm_tools::binutils::disassemble;
+use clvm_tools_rs::classic::clvm_tools::stages::stage_0::TRunProgram;
+use clvm_tools_rs::compiler::clvm;
+use clvm_tools_rs::compiler::compiler::{
+    DefaultCompilerOpts,
+    compile_file,
+    run_failure_to_compile_err
+};
+use clvm_tools_rs::compiler::comptypes::CompileErr;
+use clvm_tools_rs::compiler::fuzzer::{FuzzOldProgram, FuzzProgram};
+use clvm_tools_rs::compiler::sexp::SExp;
+use clvm_tools_rs::fuzzing::fuzzrng::FuzzPseudoRng;
+use clvm_tools_rs::classic::clvm_tools::clvmc::compile_clvm_text;
+use clvm_tools_rs::classic::clvm_tools::stages::stage_0::DefaultProgramRunner;
+
+use clvm_tools_rs::compiler::prims;
+
+fuzz_target!(|data: &[u8]| {
+    let mut rng = FuzzPseudoRng::new(data);
+    let opts = Rc::new(DefaultCompilerOpts::new(&"*prog*".to_string()));
+    let runner = Rc::new(DefaultProgramRunner::new());
+    let prim_map = prims::prim_map();
+    let mut old = false;
+
+    // Sickos: hahaha YES
```

Comment:
Why is there a loop in here? a comment might be helpful

Reply:
to keep producing programs off the random data until something breaks. I think i understand the process, but maybe not?

Reply:
the fuzzer will call your function repeatedly with different input. it will measure code coverage and guide the input in attempts to trigger a wider code coverage. So, the loop is implied in the fuzzing infrastructure.

Reply:
thanks, i had misinterpreted what was supposed to happen here.